clusterwithcoherence/README.md

```diff
@@ -0,0 +1,119 @@
+# Deploying a Java EE Application with Coherence on Azure Using a WebLogic Virtual Machine Cluster
+This demo shows how you can deploy a Java EE application with Coherence to Azure into a WebLogic cluster using a marketplace solution.
+## Setup
+
+* Install the Window Subsystem for Linux (or other Linux, we used [WSL 2 with Ubuntu 18.04 LST](https://docs.microsoft.com/en-us/windows/wsl/install-win10)).
+* Install Maven, zip, jar in your WSL
+* Download this repository somewhere in your WSL file system (easiest way might be to download as a zip and extract).
```

Comment:
"easiest way might be to download as a zip and extract" -> I would suggest to also add the commands to download and extract the repo.

Reply:
The same with cluster page